### DISCUSSION TITLE
Created postgres-pgdump-backup helm chart - any suggestions?

### DISCUSSION THREAD

**Discussion by EugenMayer** [prodrigestivill/docker-postgres-backup-local#89] [Q&A]
We created a helm chart for kubernetes for docker-postgres-backup-local - just wanted to shared that:

https://github.com/EugenMayer/helm-charts/tree/main/charts/postgres-pgdump-backup

Are there any particular suggestions on what to change?

One question somehow would be, which app-version is actually build? There seem neither be any github releases, nor tags, nor semver docker-tags (beside the pg version). It would be great to be able to reference an app version if possible - or is this simply not desired?
Currently i'am not able to pin-point any app-version here https://github.com/EugenMayer/helm-charts/blob/main/charts/postgres-pgdump-backup/Chart.yaml#L6=

Since i force the selection of the docker image anyway in https://github.com/EugenMayer/helm-charts/blob/main/charts/postgres-pgdump-backup/values.yaml#L92= one can argue that it is hard to name any app-version anyway.

For example we are uinsg `12-alpine-af916da` which is the commit-hash of the repo - so that might be the way it is desired?

**Comment by EugenMayer** [comment]
no need to keep questions open that are not going being be answered

**Comment by prodrigestivill** [comment]
Commit hash suffix where added some time ago, not sure if this solved this question.
